deps/chakrashim/src/v8handlescope.cc

```diff
@@ -54,28 +54,38 @@ HandleScope *HandleScope::GetCurrent() {
 }
 
 bool HandleScope::AddLocal(JsValueRef value) {
-  if (_count < _countof(_locals)) {
+  if (_count <= kOnStackLocals) {
```

Comment:
May need to handle the later fail case. What to do if later JsCallFunction fails so this _locals cache is already full?